content/consulting/index/index.json

```diff
@@ -415,6 +416,18 @@
             }
           ]
         },
+        {
```

Comment:
Shouldn't this change be in the clean architecture branch?

Reply:
As per Teams, I wanted to avoid conflicts by making changes to the same file in 2 different PRs